docs/developers.rst

```diff
@@ -0,0 +1,15 @@
+Hints to Developers
```

Comment:
Hi, good idea to add  that. Please a `CONTRIBUTING.rst` file to the repo root and include it in here.

The contributing file is also supported by GitHub and will be displayed when someone opens a PR.